packages/axios-extension/test/abap/message.test.ts

```diff
@@ -20,9 +21,9 @@ describe('message helpers', () => {
 
             const infoMock = (log.info = jest.fn());
             const warningMock = (log.warn = jest.fn());
-            prettyPrintMessage({ msg: JSON.stringify(msg), log, host });
-            // log main message, two messages for the full url, and each detail
-            expect(infoMock).toBeCalledTimes(4);
+            prettyPrintMessage({ msg: JSON.stringify(msg), log, host, isDest });
```

Comment:
I don't this test is valuable - not your change, the test itself already. I don't really think we need to check how often `log.info` is called.
Could we just change this one to check if infoMock is called (no matter how often) and have an additional test where we use a url matching `.dest` and check if the note has been logged?

Reply:
Thank you for the review.
Please review the new changes.